chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -162,6 +270,13 @@ class CurrentUser(
             callback: (Subscription) -> Unit
     ) = makeSingleCallback({ syncCurrentUser.subscribeToRoom(roomId, messageLimit, consumer) }, callback)
 
+    /**
+     * Subscribe to a [room] for multipart messages. You need to configure the [listeners] to listen
```

Comment:
I think we should probably just say "for messages". The `Multipart` was added so as not to break backwards compatibility, but the old methods are deprecated, this is *the* intended way to subscribe to messages.